packages/lwc/scripts/utils/rollup.js

```diff
@@ -6,34 +6,40 @@
  */
 const path = require('path');
 const rollupReplace = require('@rollup/plugin-replace');
-const swc = require('@swc/core');
+const babel = require('@babel/core');
+const terser = require('terser');
 const { generateTargetName } = require('./helpers');
 
-function compatPlugin() {
+function babelCompatPlugin() {
     return {
-        name: 'rollup-plugin-compat',
+        name: 'rollup-plugin-babel-compat',
         transform(source) {
-            const { code } = swc.transformSync(source, {
-                sourceMaps: false,
-                jsc: {
-                    target: 'es5',
-                },
+            const { code, map } = babel.transformSync(source, {
+                babelrc: false,
+                configFile: false,
+                presets: [
+                    [
+                        '@babel/preset-env',
+                        {
+                            targets: {
+                                ie: '11',
+                            },
+                            modules: false,
+                        },
+                    ],
+                ],
             });
-            return { code };
```

Comment:
Based on my analysis, we _could_ use SWC in place of Babel and just bring Terser back. But TBH I've lost a bit of confidence in SWC, and anyway I feel it's simpler to just switch our entire toolchain back to the way it worked ~1 month ago.

Reply:
@nolanlawson 

> lost a bit of confidence in SWC

Why? Can you clarify?

Reply:
@alexander-akait Having to debug IE11 put me in a bit of a grumpy mood. 🙂 

SWC is great because it's fast, but Babel/Terser have been pretty rock-solid for us in the past, so to me it might be worth waiting a few extra seconds for our build to complete.

Reply:
We really try to avoid such problems and make every possible efforts to be stability, yes, it is young, but already faster and compress code better than most of exiting solutions, if you faced with something strage/buggy/unexpected you can always open an issue and we will try to solve it ASAP

Reply:
@alexander-akait SWC is a super exciting project! Thanks for all your hard work on it. 🙂  I did go ahead and open an issue: https://github.com/swc-project/swc/issues/4515